test/unit/models/challenges.test.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable security/detect-object-injection */
+const chai = require("chai");
+const { expect } = chai;
+
+const cleanDb = require("../../utils/cleanDb");
+const firestore = require("../../../utils/firestore");
+const addUser = require("../../utils/addUser");
+
+const challengeQuery = require("../../../models/challenges");
+const challengeModel = firestore.collection("challenges");
+
+// Import fixtures
+const userDataArray = require("../../fixtures/user/user")();
+const challengeDataArray = require("../../fixtures/challenges/challenges")();
+const challengeData = challengeDataArray[0];
+
+describe("Challenges", function () {
+  afterEach(async function () {
+    await cleanDb();
+  });
+  describe("postChallenge", function () {
+    it("should add the challenge", async function () {
+      const challengeId = await challengeQuery.postChallenge(challengeData);
```

Comment:
Can we move it to `beforeEach` function as it's being used in multiple it blocks?